util/package.go

```diff
@@ -22,9 +22,35 @@ import (
 const defaultType = "integration"
 
 var CategoryTitles = map[string]string{
-	"logs":     "Logs",
-	"metrics":  "Metrics",
-	"security": "Security",
+	"aws":               "AWS",
+	"azure":             "Azure",
+	"cloud":             "Cloud",
+	"config_management": "Config management",
+	"container":         "Container",
+	"containers":        "Containers",
```

Comment:
@mostlyjason I stumbled over this one. I had somehow both, based on the integrations page I assume we only need `containers`?

Reply:
I removed it as I couldn't find it in the internal doc anymore.